_articles/evaluation_queues.md

```diff
@@ -14,7 +14,7 @@ To create a queue, you must first create a Synapse `Project` or have [edit permi
 
 Once you've created your project, navigate to it and add `/challenge` to the url (e.g. www.synapse.org/#!Synapse:syn12345/challenge). Click **Tools** in the right corner and select **Add Evaluation Queue**.
 
-<img style="width: 80%;" src="/assets/images/create_evaluation_queues.png">
+<img style="width: 40%;" src="../assets/images/create_evaluation_queues.png">
```

Comment:
Not sure you will need the `../`

Reply:
It's not strictly needed, but it allowed me to see the image in a markdown preview (without running a server)